eng/scripts/Language-Settings.ps1

```diff
@@ -18,7 +18,7 @@ function Get-AllPackageInfoFromRepo ($serviceDirectory)
   try
   {
     Push-Location $RepoRoot
-    pip install packaging==20.4 -q -I
+    pip install packaging==20.4 setuptools==44.1.1 -q -I
```

Comment:
We might want to start considering a requirements file for this if we start getting a longer list.

Reply:
Agree.